packages/tracing-internal/src/index.ts

```diff
@@ -1 +1,34 @@
-export {};
+export * from './exports';
+
+import { addExtensionMethods } from './extensions';
+import * as Integrations from './node/integrations';
+
+export { Integrations };
+
+// BrowserTracing is already exported as part of `Integrations` above (and for the moment will remain so for
+// backwards compatibility), but that interferes with treeshaking, so we also export it separately
+// here.
+//
+// Previously we expected users to import tracing integrations like
+//
+// import { Integrations } from '@sentry/tracing';
+// const instance = new Integrations.BrowserTracing();
+//
+// This makes the integrations unable to be treeshaken though. To address this, we now have
+// this individual export. We now expect users to consume BrowserTracing like so:
+//
+// import { BrowserTracing } from '@sentry/tracing';
+// const instance = new BrowserTracing();
+//
+// For an example of of the new usage of BrowserTracing, see @sentry/nextjs index.client.ts
+export const BrowserTracing = Integrations.BrowserTracing;
```

Comment:
this needs to be exported from `'./browser'` otherwise it prevents `Integrations` from being treeshaken.

`BrowserTracing` is part of the `Integrations` var as a backwards compat thing.

Reply:
🤦‍♂️